### PR TITLE
make id3 parsed json data same as safari

### PR DESCRIPTION
**Pull request by saxena-gaurav**

### CODE REVIEW

File: src/videojs-hls.js [hunk 1]
```diff
@@ -335,8 +337,9 @@ videojs.Hls.prototype.addCuesForMetadata_ = function(segmentInfo) {
     for (i = 0; i < metadata.frames.length; i++) {
       frame = metadata.frames[i];
       time = segmentOffset + ((metadata.pts - minPts) * 0.001);
-      cue = new window.VTTCue(time, time, frame.value || frame.url || '');
+      cue = new WebKitCue(time, time, frame.value || frame.url || '');
```
Comment (dmlap) [inline src/videojs-hls.js]: What happens if WebKitCue is not defined?

File: src/metadata-stream.js [hunk 2]
```diff
@@ -42,6 +42,7 @@
             break;
           }
         }
+        tag.data = tag.value;
```
Comment (dmlap) [inline src/metadata-stream.js]: Could we add [deprecation warnings](https://github.com/videojs/videojs-contrib-hls/blob/72ece371988ef4f18ef1c6b38074a1120fdffbb5/src/videojs-hls.js#L38-L43) for the old property locations so we can clean them up before too long?

**Comment by saxena-gaurav** [comment]
@dmlap : Thanks for the review. I have addressed the code review comments.

File: src/videojs-hls.js [hunk 3]
```diff
@@ -335,13 +337,43 @@ videojs.Hls.prototype.addCuesForMetadata_ = function(segmentInfo) {
...
+      cue = new Cue(time, time, frame.value || frame.url || '');
       cue.frame = frame;
+      cue.value = frame;
       cue.pts_ = metadata.pts;
       textTrack.addCue(cue);
     }
     segmentInfo.pendingMetadata.shift();
   }
+
+  if (cue && cue.frame) {
```
Comment (dmlap) [inline src/videojs-hls.js]: It appears this code will only run for the last cue created in the `for` loop above.

**Comment by dmlap** [comment]
metadata-stream is being moved into [mux.js](https://github.com/videojs/mux.js) in the upcoming release of this project. Would you mind moving your changes to that file to a PR in that project?

**Comment by saxena-gaurav** [comment]
https://github.com/videojs/mux.js/pull/21

**Comment by saxena-gaurav** [comment]
Against development branch:  https://github.com/videojs/videojs-contrib-hls/pull/395

**Comment by saxena-gaurav** [comment]
closing in favor of #395

**Pull request closed without merging**